pmd-core/src/main/java/net/sourceforge/pmd/RuleBuilder.java

```diff
@@ -0,0 +1,246 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd;
```

Comment:
move this class to a separate package, such as `net.sourceforge.pmd.rules`

Reply:
What do you think of moving RuleFactory too?

Reply:
Definitely!